README.rst

```diff
@@ -211,10 +211,12 @@ network.certkeyfile
+network.enabletls                  true                                                                                                                                                                                                                                                                                                             Whether to enable TLS for gRPC connections. When `certfile` or `certkeyfile` is specified it defaults to `true`, otherwise `false`.
 network.grpcaddr                   \:5555                                                                                                                                                                                                                                                                                                            Local address for gRPC to listen on. If empty the gRPC server won't be started and other nodes will not be able to connect to this node (outbound connections can still be made).
 network.nodedid                                                                                                                                                                                                                                                                                                                                     Specifies the DID of the organization that operates this node, typically a vendor for EPD software. It is used to identify the node on the network. If the DID document does not exist of is deactivated, the node will not start.
 network.protocols                  []                                                                                                                                                                                                                                                                                                               Specifies the list of network protocols to enable on the server. They are specified by version (1, 2). If not set, all protocols are enabled.
+network.tls.certheader                                                                                                                                                                                                                                                                                                                              Name of the HTTP header that will contain the client certificate when TLS is offloaded.
+network.tls.offload                                                                                                                                                                                                                                                                                                                                 Whether to enable TLS offloading for incoming connections. If enabled `network.tls.offload` must be configured as well.
```

Comment:
Yes, typo. There's no default because there's no defacto standard for the header name. Some reverse proxies have a hardcoded header name (Apacvhe), others (NGINX) don't.

Furthermore, it would a potential vulnerability: if the incorrect header name is configured and the reverse proxy doesn't strip it from the incoming HTTPS request (passing it to the Nuts node), it would allow an attacker to authenticate as whatever party on the network.